app/assets/javascripts/d3/merit_order_hourly_supply.coffee

```diff
@@ -32,8 +32,10 @@ D3.merit_order_hourly_supply =
         .offset("zero")
         .values((d) -> d.values)
 
-      @stackedData = @stack(@chartData[1..@chartData.length])
-      @totalDemand = [@chartData[0]]
+      lastId = @chartData.length - 1
+
+      @stackedData = @stack(@chartData[0...lastId])
+      @totalDemand = [@chartData[lastId]]
```

Comment:
Unfortunately, this needs to be repeated for *all* the Merit charts: dynamic demand curve, heat production, flexibility, etc...

Perhaps it's worth adding a general helper function somewhere (`Util`?) which will take an array of series and partition them appropriately: `{ targets: [targetSeries...], data: [dataSeries...] }`?